.github/workflows/release.yml

```
@@ -26,21 +26,8 @@ jobs:
       - name: Install dependencies
         run: PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=true pnpm i
 
-      - name: (assets) Run Semantic Release
+      - name: Run Semantic Release for all packages
         env:
           GH_TOKEN: ${{ secrets.GH_RELEASE_BUILDUSERSICKAG }}
           NPM_TOKEN: ${{ secrets.NPM_RELEASE_TOKEN_CS }}
-        run: cd packages/assets && pnpm release
-
-      - name: (components) Run Semantic Release
-        env:
-          GH_TOKEN: ${{ secrets.GH_RELEASE_BUILDUSERSICKAG }}
-          NPM_TOKEN: ${{ secrets.NPM_RELEASE_TOKEN_CS }}
-        run: cd packages/components && pnpm release
-
-      - name: (tokens) Run Semantic Release
-        env:
-          GH_TOKEN: ${{ secrets.GH_RELEASE_BUILDUSERSICKAG }}
-          NPM_TOKEN: ${{ secrets.NPM_RELEASE_TOKEN_CS }}
-        run: cd packages/tokens && pnpm release
-
+        run: pnpm build && pnpm release && pnpm publish --tag alpha
```

```suggestion
        run: pnpm build && pnpm release && pnpm publish --tag main
```

Shouldn't this be `main` to stick to the actual version name?

---

Good question. I have opted for alpha as this is the way most packages do stuff like that when using publish tags. For me (personally), it is more common ground installing package scopes via `npm install --save @synergy-design-system/components@alpha` than via `npm install --save @synergy-design-system/components@main`. But I am open for this as we do not have yet an audience ;)

---

Yah, don't have an opinion neither. It's just that we already published something with main and we're currently in prerelease pahse.... but I don't care. 😆

---

Done in latest commmit